pkg/compiler/syscall.go

```diff
@@ -35,6 +35,7 @@ var syscalls = map[string]map[string]Syscall{
 		"ECDSASecp256k1CheckMultisig": {"Neo.Crypto.CheckMultisigWithECDsaSecp256k1", false},
 		"ECDsaSecp256r1Verify":        {"Neo.Crypto.VerifyWithECDsaSecp256r1", false},
 		"ECDSASecp256r1CheckMultisig": {"Neo.Crypto.CheckMultisigWithECDsaSecp256r1", false},
+		"RIPEMD160":                   {"Neo.Crypto.RIPEMD160", false},
```

Comment:
#1193

Reply:
There was a commit description that this commit closes 1193. Isn't it enough or I should create a separate PR for this?